src/lib/components/utils/TagList.svelte

```diff
@@ -32,11 +32,11 @@
     target: 'popupAutocomplete',
     placement: 'bottom-start'
   };
-
   let tagList = [];
-  const loadTagList = () => (tagList = tags.map((t: Tag) => t.name));
+  const loadTagList = tags ? () => (tagList = tags.map((t: Tag) => t.name)) : () => {};
 
   onMount(loadTagList);
+  afterUpdate(loadTagList);
```

Comment:
This could constantly retrigger updates, because it's not a reactive change (`$:`), so setting `tagList` will trigger an update as there are reactive listeners to it (the bindings in the section HTML)